Assets/Scripts/Manager/GhostSwitchManager.cs

```diff
@@ -208,6 +211,7 @@ void possess (GameObject player, GameObject doll)
 
 		DollAudioManager.getInstance ().playGhostSwitchSound ();
 		updateCamera ();
+		updateAudio ();
```

Comment:
Again thats for when I need to update to tell audio controller that ghost switch already happen